composer.json

```diff
@@ -5,20 +5,20 @@
     "keywords": ["ace", "code", "editor", "symfony", "form", "bundle", "javascript"],
     "license": "MIT",
     "require": {
-        "php": ">=7.3",
-        "twig/twig": "^1.34|^2.0|^3.0",
-        "symfony/framework-bundle": "^3.0|^4.0|^5.0",
-        "symfony/form": "^3.0|^4.0|^5.0",
-        "symfony/twig-bridge": "^3.0|^4.0|^5.0",
-        "symfony/asset": "^3.0|^4.0|^5.0",
-        "symfony/expression-language": "^3.0|^4.0|^5.0"
+        "php": "~8.1 || ~8.2 || ~8.3",
```

Comment:
since composer is not really handling `~` properly, it should be `~8.1.0 || ~8.2.0 || ~8.3.0` - when `~8.3` is used composer will also allow for 8.4, but when `~8.3.0` is used only `php >= 8.3 and < 8.4` are allowed